terraso_backend/apps/shared_data/visualization_tileset_tasks.py

```diff
@@ -62,85 +63,105 @@ def get_owner_name(visualization):
+                "label": data_point["label"],
+                "value": row[data_point["index"]],
             }
-            for data_point in data_points
+            for data_point in data_points_indexes
         ]
 
-        annotation_title = annotate_config.get("annotationTitle")
-
-        title_index = (
-            first_row.index(annotation_title)
-            if annotation_title and annotation_title in first_row
-            else None
-        )
+        properties = {
+            "title": row[title_index] if title_index else None,
+            "fields": json.dumps(fields),
+        }
 
-        features = []
-        for row in rows:
-            fields = [
-                {
-                    "label": data_point["label"],
-                    "value": row[data_point["index"]],
-                }
-                for data_point in data_points_indexes
-            ]
-
-            properties = {
-                "title": row[title_index] if title_index else None,
-                "fields": json.dumps(fields),
+        try:
+            longitude = float(row[longitude_index])
+            latitude = float(row[latitude_index])
+            feature = {
+                "type": "Feature",
+                "geometry": {
+                    "type": "Point",
+                    "coordinates": [longitude, latitude],
+                },
+                "properties": properties,
             }
 
-            try:
-                longitude = float(row[longitude_index])
-                latitude = float(row[latitude_index])
-                feature = {
-                    "type": "Feature",
-                    "geometry": {
-                        "type": "Point",
-                        "coordinates": [longitude, latitude],
-                    },
-                    "properties": properties,
-                }
-
-                features.append(feature)
-            except ValueError:
-                continue
-
-        geojson = {
-            "type": "FeatureCollection",
-            "features": features,
-        }
+            features.append(feature)
+        except ValueError:
+            continue
+
+    return {
+        "type": "FeatureCollection",
+        "features": features,
+    }
+
+
+def _get_geojson_from_gis(data_entry):
+    file = data_entry_upload_service.get_file(data_entry.s3_object_name, "rb")
+    return parse_file_to_geojson(file)
+
+
+def _get_geojson_from_data_entry(data_entry, visualization):
+    is_dataset = f".{data_entry.resource_type}" in settings.DATA_ENTRY_SPREADSHEET_TYPES.keys()
+    is_gis = f".{data_entry.resource_type}" in settings.DATA_ENTRY_GIS_TYPES.keys()
+
+    if is_dataset:
+        return _get_geojson_from_dataset(data_entry, visualization)
+
+    if is_gis:
+        return _get_geojson_from_gis(data_entry)
+
+
+def create_mapbox_tileset(visualization_id):
+    logger.info("Creating mapbox tileset", visualization_id=visualization_id)
```

Comment:
```suggestion
```

Reply:
@paulschreiber This can't be removed since it is an async process and we ned to log this in case we need to understand what happened. Also this was not added as part of this PR